blog/20240314-simplex-chat-v5-6-quantum-resistance-signal-double-ratchet-algorithm.md

```diff
@@ -0,0 +1,260 @@
+
+### 6. "Record now, decrypt later" attacks - mitigated by post-quantum cryptography.
+
+This is the idea based on the assumption that commercially viable quantum computers will become available during the next 10 years, and then they can use time-efficient [Shor's algorithm](https://en.wikipedia.org/wiki/Shor%27s_algorithm) developed in 1994 to break asymmetric encryption with quantum computer (symmetric encryption is not vulnerable to this algorithm).
+
+Post-quantum cryptography, or encryption algorithms that are resistant to quantum computers, has been the area of ongoing research for several decades, and there are some algorithms that _might_ protect from quantum computers. It's important to account for these limitations:
+
+- _none of the post-quantum algorithms are proven to be secure_ against quantum or conventional computers. They are usually referred to as "believed to be secure" by the researchers and security experts. There is continuous research to break post-quantum algorithms, and to prove their security, and many of these algorithms are broken every year, often by conventional computers.
+- because of the lack of proofs or guarantees that post-quantum cryptography delivers on its promise, these algorithms can only be used in hybrid encryption schemes to augment conventional cryptography, and never to replace it, contrary to some expert recommendations, as DJB explains in this [blog post](https://blog.cr.yp.to/20240102-hybrid.html).
+- they are much more computationally expensive and less space efficient, and the encryption schemes have to balance their usability and security.
+- many of post-quantum algorithms have known patent claims, so any system deploying them accepts the risks of patent litigation.
+- the silver lining to all these limitations is that the risk of appearance of commercially viable quantum computers in the next decade may be strongly exaggerated, possibly to justify the current R&D funding for both computers and algorithms development.
+
+So, to put it bluntly, post-quantum cryptography is a remedy against the illness that nobody has, without any guarantee that it will work. The closest analogy in the field of medicine is _snake oil_, although, in fairness to snake oil, it was usually sold to cure the real illnesses people already had, not the future ones.
+
+<img src="./images/20240314-datacenter.jpg" width="400" class="float-to-right">
```

Comment:
unclear what the image is for (I only know it's for a "large datacenter" from a previous TODO)
```suggestion
```